.github/CONTRIBUTING.md

```diff
@@ -142,7 +149,16 @@ builder failures, you may be experiencing a [test suite
 issue](https://github.com/zfsonlinux/zfs/issues?q=is%3Aissue+is%3Aopen+label%3A%22Test+Suite%22).
 There are also various [buildbot options](https://github.com/zfsonlinux/zfs/wiki/Buildbot-Options)
 to control how changes are tested.
-* All proposed changes must be approved by a ZFS on Linux organization member.
+* If your pull request has not yet passed the test suite,
+Consider opening a draft PR for proposed changes which you would like to discuss or
+require additional testing. Once the PR is in good shape you can remove the draft
+status which allows it to be merged.
```

Comment:
I took another attempt and the phrasing, how about this softer wording which explains why you might want to create a draft PR and how that process works.

* If you have an idea you'd like to discuss or which requires additional testing consider opening it as a draft PR.  Once everything is in good shape and the details have been worked out you can remove its draft status.  Any required reviews can then be finalized and the PR merged.

Reply:
Done!